portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,33 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+import com.google.sps.data.Comments;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+/** Servlet that returns comments */
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
+  private final Comments comments = new Comments();
+  private final Gson GSONObj = new Gson();
```

Comment:
nit: make the field name camel case: https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names

Reply:
Fixed.